packages/macros/tests/babel/helpers.ts

```diff
@@ -4,14 +4,17 @@ import { allBabelVersions as allBabel, runDefault } from '@embroider/test-suppor
 import 'qunit';
 
 export { runDefault };
-
 export function allBabelVersions(createTests: (transform: (code: string) => string, config: MacrosConfig) => void) {
   let config: MacrosConfig;
 
   describe('without presets', function() {
     allBabel({
       babelConfig() {
         config = new MacrosConfig();
+        if (createTests.length < 2) {
```

Comment:
We should probably add a comment about this

Reply:
ya

Reply:
doing that now.